scripts/add-aws-account-with-instance-role.sh

```diff
@@ -1,27 +1,25 @@
 #!/bin/bash
```

Comment:
We recommend to have #!/bin/bash -e at the top of the bootstrapping scripts (but not directly in the UserData block) so that if something fails, you can trigger and exit and signal a failure to the CloudFormation stack, instead of waiting until timeout.